src/components/queryBuilder/utils.ts

```diff
@@ -259,6 +264,195 @@ export const getSQLFromQueryOptions = (options: SqlBuilderOptions): string => {
   return query;
 };
 
+export function getQueryOptionsFromSql(sql: string): SqlBuilderOptions {
+  const ast = sqlToAST(sql);
+  const fromPhrase = ast.get('FROM');
+  if (!fromPhrase || fromPhrase.length > 1 || !isString(fromPhrase[0])) {
+    return {} as SqlBuilderOptions;
+  }
+
+  const databaseAndTable = fromPhrase[0].trim().split('.');
```

Comment:
if they set a default database, then the sql query doesn't require database.table.  they can just use table

```
select foo from bar
```